[jit][fix] Remove slot in parameter slot

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #33481 [quant][graphmode] Handling ops doesn't require observation in insertObservers
* #33279 [quant][graphmode] Add quantized conv2d-relu fusion pattern
* #33277 [WIP][quant][graphmode] Observing input/output values in callsite
* #33276 [quant][graphmode][refactor] Move check for weight outside of insertObserverFor
* #33275 [quant][graphmode][refactor] Move values_to_skip check inside valueNeedsToBeQuantized
* #33274 [quant][graphmode][refactor] Simplify signature for insertObserverFor
* #33273 [quant][graphmode][refactor] Checks for bias and weight
* #33173 [quant][graphmode] Add a pass to handle ops that doesn't require observation
* #33172 [quant][graphmode][refactor] Factor out insertDequantCall
* #33171 [quant][graphmode] refactor nodeQuantizable
* #32816 [quant][graphmode] Add add_relu pattern in skip values
* #32815 [quant][graphmode] Add asserts for Conv and ReLU modules
* #32814 [quant][graphmode] Skip quantizing input and output in matched module
* #32813 [quant][graphmode][refactor] Separate preprocess step for insertObserver
* #32812 [quant][graphmode][refactor] Change signature of getModuleAccessPath
* #32809 [quant][graphmode][refactor] Move the check for qconfig inside insertObserver call
* #32379 [quant][graphmode] FoldConvBatchNorm2d support shared ClassTypes
* #32964 [jit] Add None parameter as parameter instead of attributes
* **#32846 [jit][fix] Remove slot in parameter slot**

Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

Differential Revision: [D19844711](https://our.internmc.facebook.com/intern/diff/D19844711)

jerryzh168 committed:
[jit][fix] Remove slot in parameter slot

Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 5cd20c5d:


**None of the build failures appear to be your fault**.

* **1/1** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=413c0f6c2930c96f52899069fcef94b7e1dd8be4&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base 413c0f6c since Jan 29
  <details>
  <summary>
  Please rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=5cd20c5d96fe53231d99a7be3fc2cab9c225e707).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [pytorch_linux_backward_compatibility_check_test](https://dr.pytorch.org/build-details.html?build_id=93518587) from Jan 29



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

Differential Revision: [D19844711](https://our.internmc.facebook.com/intern/diff/D19844711)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

Differential Revision: [D19844711](https://our.internmc.facebook.com/intern/diff/D19844711)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

Differential Revision: [D19844711](https://our.internmc.facebook.com/intern/diff/D19844711)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

Differential Revision: [D19844711](https://our.internmc.facebook.com/intern/diff/D19844711)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

Differential Revision: [D19844711](https://our.internmc.facebook.com/intern/diff/D19844711)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

Differential Revision: [D19844711](https://our.internmc.facebook.com/intern/diff/D19844711)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[jit][fix] Remove slot in parameter slot"


Summary:
att
Test Plan:
build/bin/test_jit
Reviewers:
suo
Subscribers:

Tasks:

Tags:

Differential Revision: [D19844711](https://our.internmc.facebook.com/intern/diff/D19844711)

[ghstack-poisoned]
